wdio.conf.js

```diff
@@ -30,6 +30,9 @@ const config = {
       }],
     },
   },
+  terra: {
```

Comment:
This should be defaulted and no changes should be needed in this file. See comment above

Reply:
I was thinking this was the default configuration. If not, I can add it back to the service file.

Reply:
This file is the wdio.config for the Toolkit repo. Here is the [default config file](https://github.com/cerner/terra-toolkit/blob/1ecb511eac01274dd31faa17e2eb9597c7665788/src/wdio/conf.js) that we provide.

Reply:
See if this works for you. I added it to the default config file. https://github.com/cerner/terra-toolkit/pull/48/commits/09ee1978cfcf0f50f51ee1f85a3ed41308da06e9